src/cocoa/toga_cocoa/widgets/canvas.py

```diff
@@ -152,6 +152,10 @@ def stroke(self, color, line_width, draw_context, *args, **kwargs):
         else:
             # Set color to black
             core_graphics.CGContextSetRGBStrokeColor(draw_context, 0, 0, 0, 1)
+        if line_dash is not None:
+            core_graphics.CGContextSetLineDash(draw_context,0,(CGFloat*len(line_dash))(*line_dash),2)
+        else:
+            core_graphics.CGContextSetLineDash(draw_context,0,None,0)
```

Comment:
Not sure why this isn't getting picked up by flake8, but the spacing around arguments is off here - there should be spaces after commas in argument lists.